runner/app/live/api/api.py

```diff
@@ -124,23 +124,39 @@ async def handle_start_stream(request: web.Request):
         config_logging(request_id=params.request_id, manifest_id=params.manifest_id, stream_id=params.stream_id)
 
         # Try to get dimensions from workflow first
-        width = params.params.get("width", DEFAULT_WIDTH)
-        height = params.params.get("height", DEFAULT_HEIGHT)
+        input_width = params.params.get("width", DEFAULT_WIDTH)
+        input_height = params.params.get("height", DEFAULT_HEIGHT)
```

Comment:
Naming nit (but naming is hard)

IIRC this originally meant the *target* (ie, output) resolution ... these params still reflect the output from the decoder, correct?

From the decoder's POV, "input" is the source stream and we don't know the resolution for that until we actually receive it. Passing in something named the "input" to the decoder for the output feels confusing.

Is this meant to be the resolution that we do inference at? Maybe something like `infer_width, infer_height`  or thereabouts?